teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(id, age, specialization, name = 'Uknown', parent_permission: true)
+    super(id, age, name, parent_permission)
```

Comment:
- [ ] Kindly remove the `id` from here. It is now auto-generated by the app.

- [ ] The `parent_permission` argument should be assigned the correct way in the `super` method argument

  ```
  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    ...
  ```